.github/workflows/ci.yml

```diff
@@ -74,6 +94,7 @@ jobs:
       run: |
         cargo nextest run \
           --workspace \
+          --exclude lichee-rv \
```

Comment:
nit, take it or leave it: maybe worth a comment explaining why this is excluded?